src/test/java/com/google/maps/DirectionsApiTest.java

```diff
@@ -394,4 +394,29 @@ public void testGeocodedWaypoints() throws Exception {
 
   }
 
+  /**
+   * Test {@code local_icon} for Directions in Paris.
+   */
+  @Test
+  public void testLocalIconInParis() throws Exception {
+    DirectionsResult result = DirectionsApi.newRequest(context)
+        .origin("paris metro bibliotheque francois mitterrand")
+        .destination("paris metro pyramides")
+        .mode(TravelMode.TRANSIT)
+        .await();
+    assertNotNull(result);
```

Comment:
I know there's not, but I'm really sad there's not a better way to do this.

Well.. actually. You could just...

```java
String localIcon = "";
try {
  localIcon = result.routes[0].legs[0].....
} catch (NullPointerException e) {
  fail("expected transit...");
}
assertEquals(...);
```

Just my 2c, up to you though.